platform/core/src/classes/ImageSet.ts

```diff
@@ -1,15 +1,11 @@
 import guid from '../utils/guid.js';
-import { Vector3 } from 'cornerstone-math';
+import calculateScanAxisNormal from '../utils/calculateScanAxisNormal';
 
 type Attributes = Record<string, unknown>;
 type Image = {
   StudyInstanceUID?: string;
-  getData(): {
-    metadata: {
-      ImagePositionPatient: number[];
-      ImageOrientationPatient: number[];
-    };
-  };
+  ImagePositionPatient?: string;
```

Comment:
This shouldn't be a string ever - it is a Point3 if defined, same with ImageOrientationPatient.